### PR TITLE
[ET-1764] Add notice and documentation for Paystack

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1764] 
[ET-1765] 

### 🗒️ Description

We're currently Partnering with Paystack and cross-promoting products. We will display a notice if the time zone for the WordPress site is set to `Lagos`, `Accra` or `Johannesburg` (Africa) and if the TEC - Paystack plugin isn't already active. It will also only be displayed within the Tickets Settings pages.

### 🎥 Artifacts 

image

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1764]: https://theeventscalendar.atlassian.net/browse/ET-1764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ET-1765]: https://theeventscalendar.atlassian.net/browse/ET-1765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codingmusician** [comment]
> Flagged some minor nitpicks — code itself looks good to go in my opinion.

Nitpicks are welcome and appreciated, @ggwicz!

File: src/Tribe/Admin/Notices.php [hunk 1]
```diff
@@ -270,4 +271,66 @@ public function maybe_display_unsupported_currency_notice() {
 			]
 		);
 	}
+
+	function maybe_display_paystack_notice() {
```
[the remark this answers is not in the document]
Comment (codingmusician) [reply src/Tribe/Admin/Notices.php:282]: Totally forgot this. Thanks!

File: src/Tribe/Admin/Notices.php [hunk 2]
```diff
@@ -270,4 +271,73 @@ public function maybe_display_unsupported_currency_notice() {
...
+	 */
+	function maybe_display_paystack_notice() {
+
+		// Bail if we aren't in Tickets > Settings.
+		if ( \Tribe\Tickets\Admin\Settings::$settings_page_id !== tribe_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if Paystack plugin is installed and activated.
+		if ( class_exists( 'paystack\tec\classes\Core' ) ) {
```
Comment (redscar) [inline src/Tribe/Admin/Notices.php:290]: I may be incorrect, but I recall hearing that when we use `class_exists` we should set autoload to false. You may want to double check on this.

Comment (codingmusician) [reply src/Tribe/Admin/Notices.php:290]: You may be right, although this is for a plugin that isn't ours. It won't hurt to add it, though.

**Review by redscar — APPROVED** [review]

**Pull request merged**